.github/workflows/push-specs.yml

```diff
@@ -0,0 +1,35 @@
+name: Push Specs
+
+on:
+  push:
+    tags:
+      - 'cardano-ledger-specs-[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  push-docs:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v18
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+            substituters = https://cache.iog.io https://cache.nixos.org/
+      - name: Build specs
+        run: nix-build -A specs -o output/spec
+      - name: Collect dereferenced docs
+        run:  |
+          mkdir -p deref
+          for d in output/* ; do
+            dr="$(basename $d)";
+            cp -L $d/*.pdf deref;
+          done
+      - name: Create specs release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "deref/*"
+
```

Comment:
@teodanciu If I am not mistaken I think we decided to push releases this repo: https://github.com/input-output-hk/formal-ledger-specifications to a `gh-pages` pranch. This way we can setup github pages and people will be able to access the pdfs with a nice url

Reply:
actually, @teodanciu and I went round in circles yesterday trying to find the best solution. The trans-repo solution involved a few things that we didn't love, the worst of which was having to tie the security to a single individual's github account, and having that token be updated once a year. so we came back to the release idea, and we discovered that we could create a stable link ("latest") to the pdfs in a release, which can be triggered by pushing a tag.

Reply:
Oh, so it will upload pdfs to the release page? That's even better. That's what I liked and advocated all along :smile:

Reply:
Indeed it will, that's exactly what this PRs does. I was about to say "wasn't it you that suggested this idea in the first place" :laughing: .